modules/editor-service/src/main/java/org/opencastproject/editor/EditorServiceImpl.java

```diff
@@ -715,8 +842,17 @@ public EditingData getEditData(final String mediaPackageId) throws EditorService
       final TrackSubData video = new TrackSubData(track.hasVideo(), videoPreview,
                         videoEnable);
 
+      final String thumbnailURI = Arrays.stream(internalPub.getAttachments())
+              .filter(attachment -> attachment.getFlavor().getType().equals(track.getFlavor().getType()))
+              .filter(attachment -> attachment.getFlavor().getSubtype().equals(getThumbnailSubtype()))
+              .map(MediaPackageElement::getURI).map(this::signIfNecessary)
+              .findAny()
+              .orElse(null);
```

Comment:
I'm concerned this `null` will cause issues in the future, but this is *probably* fine for now since nothing else is using this.  Yet.